### DISCUSSION TITLE
Compile full CSS

### DISCUSSION THREAD

**Discussion by ciarangaffey** [tailwindlabs/tailwindcss#6879] [Help] [answered]
For development purposes I would always compile the full CSS file at the beginning of a project without purging anything. Purge is enabled by default with version 3.0 so that doesn't seem to be an option anymore. How can I generate all CSS now?

**Comment by adamwathan** [comment, marked as answer]
Hey! Check out this similar discussion for more information:

https://github.com/tailwindlabs/tailwindcss/discussions/6256

**Reply by ciarangaffey** [reply]
Thank you!
